frontend/projects/valtimo/layout/src/lib/components/widget-interactive-table/widget-interactive-table-search/widget-interactive-table-search.component.scss

```
@@ -40,7 +51,44 @@
   }
 }
 
+::ng-deep .multiple-search-fields {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)